spec/models/form_handler_spec.rb

```diff
@@ -1,43 +1,85 @@
 require "rails_helper"
 
 RSpec.describe FormHandler do
-  let(:test_form_name) { "2021_2022" }
+  let(:form_handler) { described_class.instance }
+
+  before { Singleton.__init__(described_class) }
```

Comment:
What does this do/why is it needed?

Reply:
When we change time using timecop we need to reload the FormHandler instance because current_form would change to a different one in a different year and we're only initialising the forms at the very start I think.
This resets the the FormHandler

Reply:
Cool. Worth a code comment maybe? Or think it's obvious enough?